core/src/main/java/io/ddf/content/Schema.java

```diff
@@ -401,12 +401,17 @@ public void setData(Object[] data) {
    * The R concept of a column "type", such as STRING, INT, LONG, etc.
    */
   public enum ColumnType {
-
-    STRING(String.class), INT(Integer.class), LONG(Long.class), FLOAT(
-        Float.class), DOUBLE(Double.class), BIGINT(Long.class), //
-    TIMESTAMP(Date.class, java.sql.Date.class, Time.class, Timestamp.class), BLOB(
-        Object.class), //
-    LOGICAL(Boolean.class), //
+    STRING(String.class),
+    INT(Integer.class),
+    LONG(Long.class),
+    FLOAT(Float.class),
+    DOUBLE(Double.class),
+    BIGINT(Long.class), //
+    TIMESTAMP(Date.class, java.sql.Date.class, Time.class, Timestamp.class),
+    STRUCT(Object.class),
```

Comment:
Supporting a datatype means we must support operations on that datatype. For example, how can users refer to the struct's members? how can they access to an array's element?